source/_posts/2024-01-03-release-20241.markdown

```diff
@@ -0,0 +1,578 @@
+data:
+  device_id: 87dc3b85c18ee84f8c322a95e8230eb2
+  name: Front door
+  filename: test.mp4
+``` 
+
+New service call:
+```yaml
+service: blink.save_video
+target:
+  entity_id:
+    - camera.front_door
+data:
+  filename: test.mp4
+``` 
+
+---
+
+The `blink.send_pin` service now uses an integration/configuration entry selector:
+
+Example service call before 2023.11:
+
+```yaml
+service: blink.send_pin
+data:
+  pin: "1234"
+```
+
+Service call after 2023.11 and before 2024.1:
+
+```yaml
+service: blink.send_pin
+data:
+  pin: "1234"
+  device_id: 87dc3b85c18ee84f8c322a95e8230eb2
+```
+
+ New service call:
+
+```yaml
+service: blink.send_pin
+data:
+  config_entry_id: a1bee602deade2b09bc522749bbce48e
+  pin: "1234"
+```
+
+---
+
+The `blink.blink_update` service is deprecated and will be removed in 2024.7.0.
+Use the `homeassistant.update_entity` service instead.
+
+([@mkmer] - [#105413]) ([documentation](/integrations/blink))
+
+[@mkmer]: https://github.com/mkmer
+[#105413]: https://github.com/home-assistant/core/pull/105413
+
+{% enddetails %}
+
+{% details "Home Assistant shutdown event" %}
+
+The event key in the (undocumented) trigger data for the `homeassistant` trigger
+platform is now the string `shutdown` for a `homeassistant` trigger configured
+to fire on shutdown instead of an `Event` object.
+
+([@tetele] - [#91165]) ([documentation](/integrations/homeassistant))
+
+[@tetele]: https://github.com/tetele
+[#91165]: https://github.com/home-assistant/core/pull/91165
+
+{% enddetails %}
+
+{% details "Modbus" %}
+
+Setting `swap: none` has been removed from the Modbus configuration.
+It is identical to omitting it: `swap:`.
+
+([@janiversen] - [#104713]) ([documentation](/integrations/modbus))
+
+[@janiversen]: https://github.com/janiversen
+[#104713]: https://github.com/home-assistant/core/pull/104713
+
+{% enddetails %}
+
+{% details "PrusaLink" %}
+
+- The printer state now contains several new statuses, most importantly
+  "attention", which always needs your invention.
+- Firmware 4.7.0 is now the minimum version for Prusa XL and MK4.
+- Firmware 5.1.0 is now the minimum version for Prusa Mini since 4.7.x and 5.0.x
+  aren't available for the Mini.
+
+([@Skaronator] - [#103396]) ([documentation](/integrations/prusalink))
+
+[@Skaronator]: https://github.com/Skaronator
+[#103396]: https://github.com/home-assistant/core/pull/103396
+
+{% enddetails %}
+
+{% details "Reolink" %}
+
+The Reolink `Infrared lights in night mode` entity has been migrated from
+the `light` to the `switch` domain.
+
+This means the entity ID also changed, and automations and dashboards may need
+to be adjusted to use the new entity ID. 
+
+If you want to go back to the light domain, use the
+[switch_as_x "Show as" option](/integrations/switch_as_x/#change-device-type-of-a-switch-from-the-existing-entity)
+on the new `Infrared lights in night mode` switch entity and select light.
+
+([@starkillerOG] - [#104713]) ([documentation](/integrations/reolink))
```

Comment:
#104713 seems to be the wrong PR for this change.

Reply:
Correct